test/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+
+if(POLICY CMP0079)
+    cmake_policy(SET CMP0079 NEW)
+endif()
+
+include_directories(
+    ${CMAKE_SOURCE_DIR}/include
+    ${CMAKE_SOURCE_DIR}/src)
+
+add_executable(libxmp-test test.c ../src/md5.c)
+set_property(TARGET libxmp-test PROPERTY C_STANDARD 90)
+if(WIN32) # Make sure unit test will find its DLLs while running
```

Comment:
```suggestion
if(BUILD_SHARED) # Make sure unit test will find its DLLs while running
```

Reply:
This behavior is required for Windows only, on Unix-like operating systems they get the `-rpath` being set (that gets clear when you run the `install` target) that makes all built libraries been found with no matter where the executable itself is located. But, if you mean the consistency to put unit-test binaries at the same directory as others, then ok.